corehq/sql_db/tests/test_connections.py

```diff
@@ -94,16 +102,25 @@ def test_standby_filtering(self, *args):
             },
         }
         with override_settings(REPORTING_DATABASES=reporting_dbs):
-            # should always return the `other` db since `ucr` has bad replication delay
+            # should always return the `ucr` db since `other` has bad replication delay
             manager = ConnectionManager()
             self.assertEqual(
-                ['other', 'other', 'other'],
+                ['ucr', 'ucr', 'ucr'],
```

Comment:
Some of these tests used weird config because of the patching it worked but it seemed weird to filter out a primary DB so I changed this to rather filter out the standby.